backend/apps/accounts/views.py

```
@@ -525,6 +525,29 @@ def post(self, request):
         email = serializer.validated_data["email"].lower()  # type: ignore
         user = User.objects.filter(email__iexact=email).first()
 
+        try:
+            user = User.objects.get(email=email)
+            # Generate reset token
+            token = generate_reset_token(user)
```

Ruff (0.15.21)

[error] Undefined name `generate_reset_token`

(F821)